examples/notify.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python3
+
+from gi.repository import GLib
+import gi
+
+gi.require_version('Playerctl', '2.0')
+from gi.repository import Playerctl
+
+manager = Playerctl.PlayerManager()
+
+gi.require_version('Notify', '0.7')
+from gi.repository import Notify
+
+Notify.init("Media Player")
+notification = Notify.Notification.new("");
+
+from urllib.parse import urlparse, unquote
+from pathlib import Path
+from gi.repository import GdkPixbuf
+import os.path
+
+def notify(player):
+    metadata = player.props.metadata;
+    keys = metadata.keys()
+    if 'xesam:artist' in keys and 'xesam:title' in keys:
+        notification.update(metadata['xesam:title'], metadata['xesam:artist'][0])
+        path = Path(unquote(urlparse(metadata['xesam:url']).path)).parent / "cover.jpg";
```

Comment:
You'll need a check for `xesam:url` to be there.

And how do you know `cover.jpg` will be at that path?